README.md

```diff
@@ -1,74 +1,71 @@
 # Qadence
 
-*Qadence* is a Python package that provides a simple interface to build _**digital-analog quantum
-programs**_ with tunable interaction defined on _**arbitrary qubit register layouts**_.
+**Qadence** is a Python package that provides a simple interface to build _**digital-analog quantum
+programs**_ with tunable qubit interaction defined on _**arbitrary register topologies**_ realizable on neutral atom devices.
 
 [![pre-commit](https://github.com/pasqal-io/qadence/actions/workflows/lint.yml/badge.svg)](https://github.com/pasqal-io/qadence/actions/workflows/lint.yml)
 [![tests](https://github.com/pasqal-io/qadence/actions/workflows/test_fast.yml/badge.svg)](https://github.com/pasqal-io/qadence/actions/workflows/test_fast.yml)
 
 ## Feature highlights
 
 * A [block-based system](tutorials/getting_started.md) for composing _**complex digital-analog
-  programs**_ in a flexible and extensible manner. Heavily inspired by
-  [`Yao.jl`](https://github.com/QuantumBFS/Yao.jl) and functional programming concepts.
+  programs**_ in a flexible and scalable manner, inspired by the Julia quantum SDK
+  [Yao.jl](https://github.com/QuantumBFS/Yao.jl) and functional programming concepts.
 
-* A [simple interface](digital_analog_qc/analog-basics.md) to work with _**interacting qubit systems**_
-  using [arbitrary qubit registers](tutorials/register.md).
+* A [simple interface](digital_analog_qc/analog-basics.md) to work with _**interacting neutral-atom qubit systems**_
+  using [arbitrary registers topologies](tutorials/register.md).
 
-* Intuitive, [expression-based system](tutorials/parameters.md) built on top of `sympy` to construct
-  _**parametric quantum programs**_.
+* An intuitive [expression-based system](tutorials/parameters.md) developed on top of the symbolic library [Sympy](https://www.sympy.org/en/index.html) to construct _**parametric quantum programs**_ easily.
 
-* [Higher-order generalized parameter shift](link to psr tutorial) rules for _**differentiating
-  arbitrary quantum operations**_ on real hardware.
+* [High-order generalized parameter shift rules](link to psr tutorial) for _**differentiating parametrized quantum operations**_.
 
-* Out-of-the-box automatic differentiability of quantum programs using [https://pytorch.org](https://pytorch.org)
+* Out-of-the-box _**automatic differentiability**_ of quantum programs with [PyTorch](https://pytorch.org/) integration.
 
-* `QuantumModel`s to make `QuantumCircuit`s differentiable and runnable on a variety of different
-  backends like state vector simulators, tensor network emulators and real devices.
+* _**Efficient execution**_ on a variety of different purpose backends: from state vector simulators to tensor network emulators and real devices.
 
-Documentation can be found here: [https://pasqal-qadence.readthedocs-hosted.com/en/latest](https://pasqal-qadence.readthedocs-hosted.com/en/latest).
```

Comment:
we should already know the public RTD address, right? can't we just put it in in this MR?

Reply:
Thanks, done.